src/common/adbc/adbc.cpp

```diff
@@ -1269,7 +1277,7 @@ AdbcStatusCode ConnectionGetObjects(struct AdbcConnection *connection, int depth
 						LIST({
 							column_name: column_name,
 							ordinal_position: ordinal_position,
-							remarks : '',
+							remarks: '',
```

Comment:
This is just a cosmetic fix. This is not related to `table_type` support. Sorry.